rollup.config.js

```diff
@@ -139,9 +139,10 @@ async function main() {
         const input = path.join(basePath, 'src/index.js');
         const freeze = false;
 
-        // Generate the externals to use, by default don't include dependencies
-        const external = Object.keys(pkg.dependencies);
+        // Generate the externals to use, by default don't include peer dependencies
+        const external = Object.keys(pkg.peerDependencies || {});
```

Comment:
Not sure if this is correct. There are some packages (advanced-bloom) that still has a dependency and we should maintain those along with the peer dependencies.
```suggestion
        const external = [].concat(
            Object.keys(pkg.dependencies || {}),
            Object.keys(pkg.peerDependencies || {})
        );
```